libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -2437,12 +2449,9 @@ MAV_RESULT GCS_MAVLINK::handle_command_get_home_position(const mavlink_command_l
     if (!AP::ahrs().home_is_set()) {
         return MAV_RESULT_FAILED;
     }
-    send_home(AP::ahrs().get_home());
+    send_home();
+    send_ekf_origin();
```

Comment:
Don't we need to send RESULT_FAILED if these methods fail to send it?

Reply:
Yes, good point!

My message-intervals stuff makes the send functions bool, so I'll make sure this gets fixed in there.